src/Platform/Microsoft.Testing.Platform/IPC/Constants.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.IPC;
```

Comment:
pls add a !!!HUGE WARNING MESSAGE!!! here where we explain that this file needs to be kept aligned to the SDK one and that the ids !!!CANNOT BE!!! absolutely exchange

Explain a bit the protocol logic/reason for future devs